src/Tracy/Debugger.php

```diff
@@ -379,7 +379,7 @@ public static function getBar()
 	{
 		if (!self::$bar) {
 			self::$bar = new Bar;
-			self::$bar->addPanel(new DefaultBarPanel('time'));
+			self::$bar->addPanel(new DefaultBarPanel('time'), 'time');
```

Comment:
Yes, it could be better. I could also use [default logic](https://github.com/nette/tracy/blob/master/src/Tracy/Bar.php#L38) without need of own ID. However I am not sure what is better.